openwisp_radius/management/commands/base/delete_unverified_users.py

```diff
@@ -27,11 +30,14 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         days = now() - timedelta(days=int(options['older_than_days']))
         exclude_methods = str(options['exclude_methods']).split(',')
-        User.objects.filter(
+        for user in User.objects.filter(
             date_joined__lt=days,
             registered_user__isnull=False,
             registered_user__is_verified=False,
-        ).exclude(registered_user__method__in=exclude_methods).delete()
+        ).exclude(registered_user__method__in=exclude_methods):
```

Comment:
```suggestion
        ).exclude(registered_user__method__in=exclude_methods).iterator():
```
I missed this in my earlier reviews. We should consider using `iterator()` whenever we are looping over a queryset. Read about iterator here: https://docs.djangoproject.com/en/3.2/ref/models/querysets/#iterator

Reply:
Yes please.